packages/studio-ui/src/web/views/FlowBuilder/index.tsx

```diff
@@ -182,23 +182,27 @@ const FlowBuilder = (props: Props) => {
         />
       )}
 
-      {!window.USE_ONEFLOW && <SidePanelInspector />}
-
-      <div className={style.diagram}>
-        <Diagram
-          readOnly={readOnly}
-          showSearch={showSearch}
-          hideSearch={() => setShowSearch(false)}
-          handleFilterChanged={handleFilterChanged}
-          highlightFilter={highlightFilter}
-          mutexInfo={mutex}
-          ref={el => {
-            if (!!el) {
-              // @ts-ignore
-              diagram = el.getWrappedInstance()
-            }
-          }}
-        />
+      <div className={style.container}>
+        <div className={style.diagram}>
+          <Diagram
+            readOnly={readOnly}
+            showSearch={showSearch}
+            hideSearch={() => setShowSearch(false)}
+            handleFilterChanged={handleFilterChanged}
+            highlightFilter={highlightFilter}
+            mutexInfo={mutex}
+            showFlowNodeProps={props.showFlowNodeProps}
```

Comment:
This property is new ? how does that affect the ui ?

Reply:
Is this props required?

Reply:
oh, I think I forgot to delete it after some debugging